src/Containers/UpdatePassword/UpdatePassword.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import imgSrc from '../../assets/images/undraw_my_password.svg';
+import logo from '../../assets/images/logo_ah.png';
+import Input from '../../Components/Common/input/input';
+import Button from '../../Components/Common/Button/Button';
+import './UpdatePassword.scss';
+import updatePasswordAction from '../../actions/updatePassword';
+
+/**
+ * Home component
+ * @param {object} props
+ * @returns {void}
+ */
+export class UpdatePassword extends Component {
+  state = {
+    password: '',
+    confirmPassword: '',
+    errorMessage: '',
+  };
+
+  /**
+   *
+   * @param {object} e
+   * @returns {object} props
+   */
+  handleInput = e => {
+    e.preventDefault();
+    this.extractToken();
+    const { target } = e;
+    this.setState({
+      [target.name]: target.value,
+      errorMessage: '',
+    });
+  };
+
+  /**
+   *
+   * @param {object} url
+   * @returns {object} props
+   */
+  extractToken = () => {
+    const { match } = this.props;
+    return match.params.token;
+  };
+
+  /**
+   *
+   * @param {object} e
+   * @returns {object} props
+   */
+  handleSubmit = e => {
+    e.preventDefault();
+    const { password, confirmPassword } = this.state;
+    const { onUpdatePassword } = this.props;
+    if (password !== confirmPassword) {
+      this.setState({
+        errorMessage: 'Password is different from Confirm Password',
+      });
+      return false;
+    }
+    const token = this.extractToken();
+    onUpdatePassword(password, token);
+  };
+
+  /**
+   *
+   * @param {object} state
+   * @returns {object} props
+   */
+  render() {
```

Comment:
> Function `render` has 58 lines of code (exceeds 25 allowed). Consider refactoring.

I would advise modularizing this function to follow the standards. 
It would be better if you can also find a way to satisfy the continuous integration claims.
Otherwise, good job Karl!